spec/requests/api/blueprints_spec.rb

```diff
@@ -0,0 +1,166 @@
+RSpec.describe "Blueprints API" do
+  describe "GET /api/blueprints" do
+    it "lists all the blueprints with an appropriate role" do
+      blueprint = FactoryGirl.create(:blueprint)
+      api_basic_authorize collection_action_identifier(:blueprints, :read, :get)
+
+      run_get(blueprints_url)
+
+      expected = {
+        "count"     => 1,
+        "subcount"  => 1,
+        "name"      => "blueprints",
+        "resources" => [
+          hash_including("href" => a_string_matching(blueprints_url(blueprint.id)))
+        ]
+      }
```

Comment:
@abellotti there could be other ways to handle it, for instance:

```ruby
expect(response.parsed_body).to include(expected).and(include_query_result(:blueprints, 1, 1))
```

Been meaning to come back to #9731, I can't decide on the best way to proceed. If going with the above, I'd prefer a method signature of something like:

```ruby
include_query_result(:blueprints, :count => 1, :subcount => 1)
```

Can I defer to a follow up PR?